.catalog-onboard-pipeline.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: v1
+offerings:
+  - name: terraform-ibm-secrets-manager
```

Comment:
the offering name is `deploy-arch-ibm-secrets-manager` (just like you added in the json)